src/main/resources/db/sql/ticket_and_user_tables_creation.sql

```diff
@@ -0,0 +1,22 @@
+CREATE TABLE IF NOT EXISTS ticket (
+    id UUID PRIMARY KEY,
+    type VARCHAR(50) NOT NULL,
+    price_in_usd DECIMAL(10, 2),
+    concert_hall_name VARCHAR(11), -- Specific to ConcertTicket
+    event_code SMALLINT, -- Specific to ConcertTicket
+    backpack_weight_in_kg FLOAT, -- Specific to ConcertTicket
+    stadium_sector CHARACTER, -- Specific to ConcertTicket
+    time TIMESTAMP, -- Specific to ConcertTicket
+    is_promo BOOLEAN, -- Specific to ConcertTicket
+    ticket_class VARCHAR(50), -- Specific to BusTicket
+    duration VARCHAR(50), -- Specific to BusTicket
+    start_date DATE, -- Specific to BusTicket
+    CHECK (type IN ('CONCERT', 'BUS'))
+);
+
+CREATE TABLE IF NOT EXISTS "user" (
+    id UUID PRIMARY KEY,
+    role VARCHAR(50) NOT NULL,
+    ticket_id UUID UNIQUE REFERENCES ticket(id), -- Specific to Client
```

Comment:
Why in user table we have ticket_id? 
What type of relation should be here?

Reply:
Removed `ticket_id` field from `user` and added `user_id` to `ticket`, added cascade delete also. Now tickets remove after owning users deletion